doc/src/vpr/debug_aids.rst

```diff
@@ -24,3 +24,77 @@ Finally, if you need to route an FPGA whose routing architecture cannot be descr
 The router, graphics, sanity checker, and statistics routines all work only with a graph that defines all the available routing resources in the FPGA and the permissible connections between them.
 If you change the routines that build this graph (in ``rr_graph*.c``) so that they create a graph describing your FPGA, you should be able to route your FPGA.
 If you want to read a text file describing the entire routing resource graph, call the ``dump_rr_graph`` subroutine.
+
+Placer and Router Debugger
+==========================
+
+.. image:: https://github.com/verilog-to-routing/verilog-to-routing.github.io/blob/master/img/debuggerWindow.png
+    :align: center
+
+Adding a breakpoint
+~~~~~~~~~~~~~~~~~~~
+
+Currently the user is required to have graphics on in order to set breakpoints. By clocking the “Debug” button, the debugger window opens up and from there the user can enter integer values in the entry fields and set breakpoints. A more advanced option is using expressions which allows a wider variety of settings since the user can incorporate multiple variables and use boolean operators. This option is found by clicking the “Advanced” button in the debugger window. Using an expression is more accurate than the entry fields when setting multiple breakpoints.
```

Comment:
Give an example early on: e.g. you might want the placer to stop after a certain number of temperatures, or the router to stop before routing a certain net so you could examine the state of the circuit at that point.

Reply:
I added an overview section to give an into and include an example